docs/api-diffs/9.0/rc.1/diff.md

```diff
@@ -0,0 +1,1290 @@
++            public static IResourceBuilder<RabbitMQServerResource> AddRabbitMQ(this IDistributedApplicationBuilder builder, [ResourceNameAttribute] string name, IResourceBuilder<ParameterResource>? userName = null, IResourceBuilder<ParameterResource>? password = null, int? port = default(int?));
+         }
+     }
+ }
+```
+
+## Aspire.Hosting.Redis
+
+``` diff
+ {
+     namespace Aspire.Hosting {
+         public static class RedisBuilderExtensions {
+-            public static IResourceBuilder<RedisResource> AddRedis(this IDistributedApplicationBuilder builder, string name, int? port = default(int?));
++            public static IResourceBuilder<RedisResource> AddRedis(this IDistributedApplicationBuilder builder, [ResourceNameAttribute] string name, int? port = default(int?));
++            public static IResourceBuilder<RedisInsightResource> WithHostPort(this IResourceBuilder<RedisInsightResource> builder, int? port);
++            public static IResourceBuilder<RedisResource> WithRedisInsight(this IResourceBuilder<RedisResource> builder, Action<IResourceBuilder<RedisInsightResource>>? configureContainer = null, string? containerName = null);
+         }
+     }
+     namespace Aspire.Hosting.Redis {
++        public class RedisInsightResource : ContainerResource {
++            public RedisInsightResource(string name);
++            public EndpointReference PrimaryEndpoint { get; }
++        }
+     }
+ }
+```
+
+## Aspire.Hosting.SqlServer
+
+``` diff
+ {
+     namespace Aspire.Hosting {
+         public static class SqlServerBuilderExtensions {
+-            public static IResourceBuilder<SqlServerDatabaseResource> AddDatabase(this IResourceBuilder<SqlServerServerResource> builder, string name, string? databaseName = null);
++            public static IResourceBuilder<SqlServerDatabaseResource> AddDatabase(this IResourceBuilder<SqlServerServerResource> builder, [ResourceNameAttribute] string name, string? databaseName = null);
+-            public static IResourceBuilder<SqlServerServerResource> AddSqlServer(this IDistributedApplicationBuilder builder, string name, IResourceBuilder<ParameterResource>? password = null, int? port = default(int?));
++            public static IResourceBuilder<SqlServerServerResource> AddSqlServer(this IDistributedApplicationBuilder builder, [ResourceNameAttribute] string name, IResourceBuilder<ParameterResource>? password = null, int? port = default(int?));
+         }
+     }
+ }
+```
+
+## Aspire.OpenAI
+
+``` diff
+ {
++    namespace Aspire.OpenAI {
++        public sealed class OpenAISettings {
++            public OpenAISettings();
++            public bool DisableMetrics { get; set; }
++            public bool DisableTracing { get; set; }
++            public Uri Endpoint { get; set; }
```

Comment:
This is 

```
public Uri? Endpoint { get; set; }
```

in the code. Is there a reason this doesn't have `?` on it?

Reply:
likely a bug in asmdiff.